[2024-09-22T02:48:33.298+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T02:48:33.377+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_table_dag.create_sum_table scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-22T02:48:33.405+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_table_dag.create_sum_table scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-22T02:48:33.406+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-22T02:48:33.460+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_sum_table> on 2024-09-21 00:00:00+00:00
[2024-09-22T02:48:33.477+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=46) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T02:48:33.480+0000] {standard_task_runner.py:72} INFO - Started process 48 to run task
[2024-09-22T02:48:33.486+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'create_table_dag', 'create_sum_table', 'scheduled__2024-09-21T00:00:00+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/create_table_dag.py', '--cfg-path', '/tmp/tmpyisw9dhe']
[2024-09-22T02:48:33.490+0000] {standard_task_runner.py:105} INFO - Job 260: Subtask create_sum_table
[2024-09-22T02:48:34.148+0000] {task_command.py:467} INFO - Running <TaskInstance: create_table_dag.create_sum_table scheduled__2024-09-21T00:00:00+00:00 [running]> on host 6e2558f7c62d
[2024-09-22T02:48:34.426+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='create_table_dag' AIRFLOW_CTX_TASK_ID='create_sum_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-21T00:00:00+00:00'
[2024-09-22T02:48:34.448+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T02:48:34.462+0000] {sql.py:266} INFO - Executing: drop table if exists SUM;

create table SUM (
    cust_id varchar PRIMARY KEY,
    count decimal,
    sum numeric(2)
);
[2024-09-22T02:48:34.484+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-22T02:48:34.508+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-22T02:48:34.519+0000] {sql.py:509} INFO - Running statement: drop table if exists SUM;

create table SUM (
    cust_id varchar PRIMARY KEY,
    count decimal,
    sum numeric(2)
);, parameters: None
[2024-09-22T02:48:34.704+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T02:48:34.707+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=create_table_dag, task_id=create_sum_table, run_id=scheduled__2024-09-21T00:00:00+00:00, execution_date=20240921T000000, start_date=20240922T024833, end_date=20240922T024834
[2024-09-22T02:48:34.749+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T02:48:34.777+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
